ansible/group_vars/all

```diff
@@ -106,7 +106,7 @@ nginx_vhosts:
         fastcgi_param PATH_INFO $path_info;
         fastcgi_param PATH_TRANSLATED $document_root$path_info;
         include snippets/fastcgi-php.conf;
-        fastcgi_pass unix:/var/run/php5-fpm.sock;
+        fastcgi_pass {{ fastcgi_pass_sock }};
```

Comment:
We could, but these are defaults configured in the PHP config files; probably /etc/php5/fpm/php-fpm.conf and /etc/php/7.0/fpm/php-fpm.conf.

IMHO, overriding a single variablein the vhosts conf that _we're already generating entirely within Ansible_ makes a lot more sense than overriding a default PHP config that we don't have under Ansible control at all yet.

Thoughts?